plugins/producer_plugin/producer_plugin.cpp

```diff
@@ -1515,32 +1507,26 @@ producer_plugin_impl::start_block_result producer_plugin_impl::start_block() {
                int num_processed = 0;
                auto unapplied_trxs_size = _unapplied_transactions.size();
                auto calculate_transaction_category = [&](const transaction_metadata_ptr& trx) {
```

Comment:
The `calculate_transaction_category` lambda is no longer necessary.

Reply:
Removed